libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1233,7 +1277,7 @@ bool AP_InertialSensor::calibrate_trim(float &trim_roll, float &trim_pitch)
     }
     level_sample /= num_samples;
 
-    if (!_calculate_trim(level_sample, trim_roll, trim_pitch)) {
+    if (!_calculate_trim(level_sample, trim_rad)) {
         goto failed;
```

Comment:
a existing issue but failed is just a `return false;`  why use a `goto`?

Reply:
fixed